js/ui/grid_core/ui.grid_core.column_fixing.js

```diff
@@ -243,6 +243,14 @@ const baseFixedColumns = {
         return $cell;
     },
 
+    _getContent: function(isFixedTableRendering) {
+        if(isFixedTableRendering) {
+            return this._fixedTableElement?.parent();
+        }
+
+        return this.callBase.apply(this, arguments);
+    },
```

Comment:
Minor comment.

Maybe the ternary operator will be better here?
```suggestion
    _getContent: function(isFixedTableRendering) {
       return isFixedTableRendering
         ? this._fixedTableElement?.parent()
         : this.callBase.apply(this, arguments);
    },
```